Kernel/API/POSIX/sys/uio.h

```diff
@@ -12,6 +12,9 @@
 extern "C" {
 #endif
 
+// Arbitrary pain threshold.
+#define IOV_MAX 1024
```

Comment:
why are you changing the value of this by a factor of 1000? Is there any reason, or is that just what it is on other systems?

Reply:
I originally added the `IOV_MAX` definition only, based on what other systems use for that value. Only later I saw that it should actually be in use, and replaced the checks inside `readv` and `writev`.

That said, I actually did misread the existing check for `KiB`, thought "what a coincidence", and left it at 1024. I have no hard feelings towards changing it to match the original restriction, but I feel like this is a more reasonable limit anyways.